src/taipy/config/checker/_checkers/_gLobal_config_checker.py

```diff
@@ -24,11 +24,19 @@ def __init__(self, config: _Config, collector: IssueCollector):
     def _check(self) -> IssueCollector:
         global_config = self._config._global_config
         self._check_clean_entities_enabled_type(global_config)
+        self._check_repository_type(global_config)
         return self._collector
 
+    def _check_repository_type(self, global_config: GlobalAppConfig):
+        value = global_config.repository_type
+        if value != "filesystem" and value != "sql":
```

Comment:
```suggestion
        if value not in ["filesystem", "sql"]:
```
for better expandability with more repository types.

Reply:
We can also move `["filesystem", "sql"]` out of the method and be a constant attribute :D